operator/config/overlays/development/manager_related_image_patch.yaml

```
@@ -9,6 +9,10 @@ spec:
         - name: manager
           env:
           - name: RELATED_IMAGE_LOKI
-            value: docker.io/grafana/loki:2.6.0
+            # Replace with next official release, e.g. v2.6.100
+            # when https://github.com/grafana/loki/pull/6659
+            # is officially released.
+            # The following tag includes just v2.6 and PR 6659:
+            value: docker.io/grafana/loki:main-2e3dfc3-amd64
```

There's also a tag without an explicit architecture set:

```suggestion
            value: docker.io/grafana/loki:main-2e3dfc3
```